src/popup/components/Filters.vue

```diff
@@ -43,7 +43,7 @@ export default {
   },
   methods: {
     handleClick([filter, { rotated = null }]) {
-      const appContainer = document.querySelector('.app-inner');
+      const appContainer = document.querySelector('.scrollable-inner');
```

Comment:
I think this is a terrible idea to match a class name of a reusable element. This can lead to major problems. I strongly suggest to use different solution for this.